misc/python/materialize/mzcompose/services/materialized.py

```diff
@@ -31,6 +31,26 @@
 from materialize.mzcompose.services.postgres import METADATA_STORE
 
 
+class MaterializeEmulator(Service):
+    """Just the Materialize Emulator with its defaults unchanged"""
+
+    def __init__(self, image: str | None = None):
+        name = "materialized"
+
+        config: ServiceConfig = {
+            "mzbuild": name,
+            "ports": [6875, 6876, 6877, 6878, 6880, 6881, 26257],
```

Comment:
We shouldn't need 6880 or 6881 anymore. The environmentd balancer ports are no longer listened on.

Reply:
Cool, will remove it separately: https://github.com/MaterializeInc/materialize/pull/30985